src/ngraph/runtime/dynamic/dynamic_backend.cpp

```diff
@@ -178,8 +178,8 @@ bool runtime::dynamic::DynamicExecutable::call(
 
     pass::Manager passes;
     passes.register_pass<pass::ConstantFolding>();
-    passes.register_pass<pass::DynElimination>();
     passes.register_pass<pass::Opset0Downgrade>(); // Converts dynamic v1 variants to v0 ops
+    passes.register_pass<pass::DynElimination>();
```

Comment:
Is this required?

Reply:
You are right, thank you, I confirm that order (from your `Broadcast` PR):
- ConstantFolding
- DynElimination
- Opset0Downgrade
works here too ;)